osdk/rustfmt.toml

```diff
@@ -0,0 +1,8 @@
+# This file contains the same content as the rustfmt.toml for Asterinas, but some configurations are commented out 
+# since they are unstable features that can only be used in the nightly channel. However, OSDK itself supports the 
+# stable channel and cannot use unstable features. Once some configurations become stable, the corresponding lines 
+# can be uncommented.
```

Comment:
I polish some wording for improved clarity. I also format it with semantic line breaks. Check out the version below.

```
# This file contains the same content as the rustfmt.toml in the root directory,
# but configurations that rely on nightly Rust are commented out.
# This is needed because unlike the rest of the project,
# OSDK is built with the stable Rust and thus cannot use unstable features.
# Once these rustfmt configurations become stable,
# the corresponding lines can be uncommented.
```